### DISCUSSION TITLE
1.19.2 Crashing on startup

### DISCUSSION THREAD

**Discussion by MrNightmare03** [shedaniel/RoughlyEnoughItems#1068] [Q & A] [answered]
[crash-2022-08-10_21.00.12-client.txt](https://github.com/shedaniel/RoughlyEnoughItems/files/9303215/crash-2022-08-10_21.00.12-client.txt)

Game crashes after the loading bar reaches 100%

**Comment by shedaniel** [comment, marked as answer]
REI is not compatible with MoreOverlays, and actually contains Builtin Inventory Highlighting, other features can be installed via different mods like Light Overlay.
